src/firefly/java/edu/caltech/ipac/table/DataType.java

```diff
@@ -286,6 +303,16 @@ public boolean isKnownType() {
         );
     }
 
+    /**
+     * Convert a string into an object
+     * Java unescaping is applied to all strings
+     * @param s string representation of an object
+     * @return an object
+     */
+    public Object convertStringToDataUnescaped(String s) {
```

Comment:
IMO: The naming is a bit confusing.  The `Unescaped` is an internal process that may be confusing for people who do not know that you use `convertStringToDataUnescaped` with `formatDataEscaped`.

I suggest overloading the `formatData` and `convertStringToData` with an additional flag `useEscape` or something similar to indicate that escape/unescape is applied to the data.. i.e

```
public String formatData(Object value, boolean useEscape)
public Object convertStringToData(String s, boolean useEscape)
```